catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/NitfHeaderAttribute.java

```diff
@@ -359,6 +369,28 @@ private NitfHeaderAttribute(String longName, String shortName,
         ATTRIBUTES.add(this);
     }
 
+    private static String convertFormat(final FileType format) {
+        if (format == FileType.NITF_TWO_ONE || format == FileType.NITF_TWO_ZERO) {
+            return NITF;
+        } else if (format == FileType.NSIF_ONE_ZERO) {
+            return NSIF;
+        }
+
+        return NITF;
```

Comment:
Because parsing would have failed, you'll never see either of the proposed values.  I would argue that "" and UNKNOWN are equally misleading to anyone reading this code.  The IllegalStateException would also never be seen, but it would be clearer to other developers that we never expect to be in that situation.